src/main/java/io/github/thebusybiscuit/slimefun4/core/services/UpdaterService.java

```diff
@@ -24,11 +25,12 @@
     private final SlimefunPlugin plugin;
     private final Updater updater;
     private final SlimefunBranch branch;
+    private final int buildNum;
```

Comment:
Can also just do updater#getVersion() instead all of this.

Reply:
Doesn't that just return the entire thing... I want specifically the build nymber

Reply:
No it returns the build number...

Reply:
https://github.com/TheBusyBiscuit/CS-CoreLib2/blob/master/src/main/java/io/github/thebusybiscuit/cscorelib2/updater/GitHubBuildsUpdater.java#L59

Reply:
My code is still better and returns int... just merge :^) <3

I want to sleep

Reply:
Definitely not better, it's just redundant. Heck it doesn't even compile >.>

Reply:
it does compile, it's stupid tests failing

Reply:
Still, you should definitely fix this before it's gonna be merged...